server/apps/publish/content/correct.py

```diff
@@ -26,6 +26,7 @@ def on_update(self, updates, original):
         ArchiveCropService().validate_multiple_crops(updates, original)
```

Comment:
Looks like your superdesk-core is pointing to old SHA. ArchiveCropService has been moved to superdesk-core as CropService.